### PR TITLE
examples/gnrc_border_router: README fixes

### PR DESCRIPTION
**Pull request by cgundogan**

### CODE REVIEW

File: examples/gnrc_border_router/README.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ run on an Atmel SAM R21 Xplained Pro evaluation board using an external UART
 adapter for the second serial interface. However, it is feasible to run the
 example on any RIOT supported platform that offers either more than one UART or
 be equipped with an IPv6 capable network device. In this case only the Makefile
-of this application has to be slightly modified (e.g. by replacing the line
+of this application has to be slightly modified, e.g. by replacing the line
 ```
 USEMODULE += gnrc_slip
 ```
-by something like
+with something like
```
Comment (OlegHahm) [inline examples/gnrc_border_router/README.md]: Aren't both prepositions correct?

Comment (cgundogan) [inline examples/gnrc_border_router/README.md]: no idea about the correctness, but it sounds strange. `...by replacing x with y` sounds IMO much better than `...by replacing x by y`.

Comment (OlegHahm) [inline examples/gnrc_border_router/README.md]: Ok, would you mind to open the PR also against the release branch?

Comment (kaspar030) [inline examples/gnrc_border_router/README.md]: dict.leo.org says "replace by so/sth" -> ablösen, "replace with so/sth" -> austauschen, auswechseln. So in this case "with" is probably correct.

Comment (OlegHahm) [inline examples/gnrc_border_router/README.md]: I asked `dict.cc` for `ersetzen durch` and it showed both options, but getting rid of the word duplication sounds good.

**Comment by OlegHahm** [comment]
ACK

**Pull request merged**